src/components/GetDataDialog.tsx

```diff
@@ -38,6 +45,10 @@ function GetDataDialog({ open, onClose }: { open: boolean; onClose: () => void }
     setShowPopover(false);
   };
 
+  const handleChange = (_event: React.MouseEvent<HTMLElement>, newCommand: string) => {
```

Comment:
I think `handleChange` is a bit generic

```suggestion
  const handleToggle = (_event: React.MouseEvent<HTMLElement>, newCommand: string) => {
```